### PR TITLE
Adds the BlinkNG post

### PR DESCRIPTION
**Pull request by rachelandrew**
Another post in the RenderingNG series.

### CODE REVIEW

File: site/en/blog/blinkng/index.md
Comment (jeffposnick) [inline site/en/blog/blinkng/index.md:183]: `hread` => `thread`

Comment (jeffposnick) [inline site/en/blog/blinkng/index.md:183]: Consider linking to the `/preview` instead of `/edit` version of the doc: https://docs.google.com/document/d/1BfIqMcJvmMFdk8e_KKeOEofz16610Mql6gmfeFhrsZk/preview

**Review by jeffposnick — DISMISSED** [review]

File: site/_data/authorsData.json
Comment (szager-chromium) [inline site/_data/authorsData.json:639]: My last name is misspelled, here and elsewhere: should be Zager

File: site/en/blog/blinkng/index.md
Comment (szager-chromium) [inline site/en/blog/blinkng/index.md:53]: Change the description to:

Once a stage has finished, its outputs should be immutable for the remainder of the rendering update.

File: site/en/blog/blinkng/index.md
Comment (szager-chromium) [inline site/en/blog/blinkng/index.md:78]: Italicize paint

File: site/en/blog/blinkng/index.md
Comment (szager-chromium) [inline site/en/blog/blinkng/index.md:115]: I think layout** is meant to be _layout_

File: site/en/blog/blinkng/index.md
Comment (szager-chromium) [inline site/en/blog/blinkng/index.md:121]: Join this paragaph with the next.

File: site/en/blog/blinkng/index.md
Comment (szager-chromium) [inline site/en/blog/blinkng/index.md:127]: remove "or" in "similar geometrical or questions were raised."

File: site/en/blog/blinkng/index.md
Comment (szager-chromium) [inline site/en/blog/blinkng/index.md:159]: "container queries requires containment" should be "container queries require containment".

File: site/en/blog/blinkng/index.md
Comment (rachelandrew) [reply site/en/blog/blinkng/index.md:24]: I think that would have been @una

File: site/en/blog/blinkng/index.md
Comment (rachelandrew) [reply site/en/blog/blinkng/index.md:158]: I couldn't find that phrase, "The reason is that an implementation of container queries requires very careful understanding and control of the relationship between the style and layout code." seems right.

**Comment by rachelandrew** [comment]
@szager-chromium there are a couple of comments above for you - would be great to get this merged!

File: site/en/blog/blinkng/index.md
Comment (szager-chromium) [reply site/en/blog/blinkng/index.md:158]: You're right, my mistake.

File: site/en/blog/blinkng/index.md
Comment (szager-chromium) [reply site/en/blog/blinkng/index.md:24]: I'll follow up with Una, thanks.

**Review by szager-chromium — COMMENTED** [review]
I don't have "approve" permissions, but this lgtm pending resolution of the illustration.

File: site/en/blog/blinkng/index.md
Comment (una) [reply site/en/blog/blinkng/index.md:24]: Untitled_Artwork

Attached the illustration, still awaiting feedback before coloring :)

Comment (szager-chromium) [reply site/en/blog/blinkng/index.md:24]: This looks good to me, my only thought is that it would be fun if you could somehow give a subtle nod to this meme template:

https://memegen.corp.google.com/384765

Comment (una) [reply site/en/blog/blinkng/index.md:24]: Updated illustration:
EA095E8E-5C1D-4A0D-962B-B6813C7FD93A

d.c.c upload url:
```
{% Img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/eg6i9S7UHWfrkF9Oube7.png", alt="ALT_TEXT_HERE", width="800", height="391" %}
```

Comment (chrishtr) [reply site/en/blog/blinkng/index.md:24]: Looks good to me also, let's add the image and publish.

File: site/en/blog/blinkng/index.md
Comment (rachelandrew) [inline site/en/blog/blinkng/index.md:24]:
```suggestion
{% Img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/eg6i9S7UHWfrkF9Oube7.png", alt="The rendering pipeline before and after BlinkNG.", width="800", height="362" %}
```

File: site/en/blog/blinkng/index.md
Comment (rachelandrew) [inline site/en/blog/blinkng/index.md:6]:
```suggestion
thumbnail: image/HodOHWjMnbNw56hvNASHWSgZyAf2/eg6i9S7UHWfrkF9Oube7.png
```

File: site/en/blog/blinkng/index.md
Comment (rachelandrew) [inline site/en/blog/blinkng/index.md:8]:
```suggestion
date: 2022-04-19
```

**Comment by rachelandrew** [comment]
@szager-chromium @chrishtr merged, so should be live in an hour or so. Once it's up I'll queue up a tweet.

**Pull request merged**
